src/main/script/ExiftoolInstall.java

```diff
@@ -32,7 +32,7 @@
  */
 public class ExiftoolInstall {
 
-    private static final Path PROJECT_ROOT = Paths.get("../../../");
+    private static final Path PROJECT_ROOT = Paths.get("");
```

Comment:
I am surprised these two statements both work :/

Reply:
I changed the path here because I change how the script was invoked. It's now invoked from the project root